felix/daemon/bootstrap_linux.go

```diff
@@ -0,0 +1,17 @@
+package daemon
```

Comment:
Please fix up the copyright/apache header.

Reply:
this is new territory for me, will try and make this a habit!